src/Assets/build/Microsoft.AspNetCore.App.Internal.Assets.targets

```diff
@@ -40,7 +40,10 @@
       <_FrameworkStaticWebAssetCandidate Remove="@(_MissingFrameworkStaticWebAssetCandidate)" />
     </ItemGroup>
 
-    <Message Importance="High" Text="Framework asset '%(_MissingFrameworkStaticWebAssetCandidate.Identity)' could not be found and won't be included in the project." />
+    <Message
```

Comment:
This is unrelated to the main fix - it was logging a message even when `_MissingFrameworkStaticWebAssetCandidate` was empty.